migrations/create_db.sql

```diff
@@ -1,5 +1,10 @@
 DROP DATABASE IF EXISTS spark;
 
+# Implicitly create the user and then drop the user.
+GRANT USAGE ON *.* TO 'spark'@'localhost' IDENTIFIED BY 'password';
+DROP USER 'spark'@'localhost';
+flush privileges;
```

Comment:
yes, but why do we need to do that? I don't strongly object to it, just that I rather let whomever is calling this script to deal with users permissions
(e.g. on travis the user is `root` and `password` is empty - no need to add a different user)